webpack.config.js

```diff
@@ -12,27 +12,20 @@ const pathToOwnWebpackConfig = join(context, "webpack.config.js");
 const hasOwnWebpackConfig = existsSync(pathToOwnWebpackConfig);
 const webpackConfigToMerge = hasOwnWebpackConfig ? require(pathToOwnWebpackConfig) : {};
 
-/**
- * Return the current version of the app.
- *
- * Format: `${package.json:version}-${gitShortSha}`
- * or just the package version if git is not available.
- */
+function getGitShortSha() {
```

Comment:
One final change and we can merge: let’s make this an arrow function for consistency.